docs/included.md

```diff
@@ -137,7 +137,15 @@
   -
     [API](api.md#scores.probability.adjust_fcst_for_crps)
   - 
-    [Tutorial](project:./tutorials/CRPS_for_CDFs.md)
+    &mdash;
+  - 
+    &mdash;
+* -  
+    - CRPS step threshold weights
```

Comment:
Most of the other 
```suggestion
    - CRPS Step Threshold Weights
```
For consistency with capitalisation in the rest of the table.

Reply:
Also, in terms of alphabetical order, it looks like CRPS Step Threshold Weights should go beneath "CRPS CDF Brier Decomposition". 

However, this section is the one place (I think) in the included.md page where we have slightly broken alphabetical order; by putting "CRPS for CDFs" before "Adjust Forecast for CRPS" - which isn't correct from an alphabetical perspective, but I think was deemed a useful way to order it thematically (so to speak).

So if you think CRPS Step Threshold Weights should be left in its current spot (between "Adjust Forecast for CRPS" and "CRPS for Brier Decomposition") - despite it not being alphabetical order - then no worries.

But if you think it would be better to use alphabetical order, then I think it should be relocated (to below "CRPS CDF Brier Decomposition"). (I hope my message makes sense).

Reply:
Another option would be tweak the CRPS for CDF (and possibly the CRPS for ensembles) section in the probability table, if there would be a better way to order/present those functions in the table.